UIforETW/Utility.cpp

```diff
@@ -431,108 +859,116 @@ bool Is64BitWindows()
 #else
 	// http://blogs.msdn.com/b/oldnewthing/archive/2005/02/01/364563.aspx
 	BOOL f64 = FALSE;
-	bool bIsWin64 = IsWow64Process(GetCurrentProcess(), &f64) && f64;
+	const bool bIsWin64 = ( ::IsWow64Process(::GetCurrentProcess(), &f64) && f64 );

```

Comment:
Cool.

I thought that the '=' statement was also extremely powerful - since it would be madness if it wasn't. Anyway, not a big deal, but it was sufficiently unusual to make it harder for me to read. And now it is zero deal.